app/javascript/components/pame/Row.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="table__row">
+    <p class="table__cell">{{ item.name }}</p>
+    <p class="table__cell">{{ item.designation }}</p>
+    <p class="table__cell">
+      <template v-if="item.restricted">{{ item.wdpa_id }}</template>
+      <a v-else :href="wdpaUrl(item.wdpa_id)" title="View protected area on Protected Planet" target="_blank">{{ item.wdpa_id }}</a>
+    </p>
+    <p class="table__cell">{{ item.id }}</p>
+    <p class="table__cell">{{ checkForMultiples('iso3') }}</p>
+    <p class="table__cell">{{ item.methodology }}</p>
+    <p class="table__cell">{{ item.year }}</p>
+    <p 
+      v-html="assessmentUrl(item.url)"
+      class="table__cell"
+    />
+    <p 
+      @click="openModal()" 
+      class="table__cell modal__trigger"
+    >
+      {{ item.metadata_id }}
+    </p>
+  </div>
+</template>
+
+<script>
+  import { eventHub } from '../../vue.js'
+
+  export default {
+    name: "row",
+    props: {
+      item: {
+        required: true,
+        type: Object,
+      }
+    },
+
+    computed: {
+      projectTitle () {
+        return this.trim(this.item.title)
+      }
+    },
+
+    methods: {
+      wdpaUrl (wdpaId) {
+        return `https://protectedplanet.net/${wdpaId}`
```

Comment:
@stacytalbot @benkt2 This should better be a URL constructed based on the environment. Probably something the backend che pass with the rest of the data

Reply:
Yeah happy to do that

Reply:
@stacytalbot I've done both FE and BE now